### PR TITLE
[KYUUBI #1868] Upgrate netty from 4.1.68 to 4.1.73

### PR DESCRIPTION
**Pull request by ulysses-you**
### _Why are the changes needed?_

close https://github.com/apache/incubator-kyuubi/issues/1868

### _How was this patch tested?_
Pass CI

### CODE REVIEW

**Comment by ulysses-you** [comment]
the dependency list changed since https://github.com/netty/netty/pull/11732

**Comment by ulysses-you** [comment]
cc @pan3793 @yaooqinn

File: externals/kyuubi-flink-sql-engine/pom.xml [hunk 1]
```diff
@@ -165,7 +165,7 @@
               <include>com.google.guava:failureaccess</include>
               <include>com.google.guava:guava</include>
               <include>commons-codec:commons-codec</include>
-              <include>io.netty:netty-all</include>
+              <include>io.netty:*</include>
```
Comment (yaooqinn) [inline externals/kyuubi-flink-sql-engine/pom.xml:168]: why do we have netty here？

Comment (ulysses-you) [reply externals/kyuubi-flink-sql-engine/pom.xml:168]: the artifact id of netty changed since netty does not provide a fat package

Comment (yaooqinn) [reply externals/kyuubi-flink-sql-engine/pom.xml:168]: where do we use netty，especially between server and engine

Comment (ulysses-you) [reply externals/kyuubi-flink-sql-engine/pom.xml:168]: The obvious place is `KyuubiConf`, it uses `NettyUtils.MAX_NETTY_THREADS`.

The netty dependency is introduced for mysql frontend.  So I think we can remove the netty dependency at engine side ? @pan3793

Comment (yaooqinn) [reply externals/kyuubi-flink-sql-engine/pom.xml:168]: we can remove it from common

Comment (yaooqinn) [reply externals/kyuubi-flink-sql-engine/pom.xml:168]: in this pr，let's do it your way

Comment (pan3793) [reply externals/kyuubi-flink-sql-engine/pom.xml]: Move `NettyUtils` to `kyuubi-server` module makes sense to me.

**Review by yaooqinn — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**